forge/src/main/java/io/github/flemmli97/flan/forge/forgeevent/ServerEvents.java

```diff
@@ -12,12 +13,16 @@
 import net.minecraftforge.event.entity.player.PlayerEvent;
 import net.minecraftforge.event.server.ServerAboutToStartEvent;
 import net.minecraftforge.event.server.ServerStartedEvent;
+import net.minecraftforge.fml.ModList;
 
 public class ServerEvents {
 
     public static void serverStart(ServerAboutToStartEvent event) {
         Flan.lockRegistry(event.getServer());
         ConfigHandler.serverLoad(event.getServer());
+
+        if (ModList.get().isLoaded("bluemap"))
```

Comment:
Might be better to move this into the mod constructor like dynmap. Is there a reason why its in this event?